appservice/src/deploy/localGitDeploy.ts

```diff
@@ -74,7 +74,7 @@ export async function localGitDeploy(client: SiteClient, fsPath: string, context
                         // for whatever reason, is '-f' exists, true or false, it still force pushes
```

Comment:
Might be able to fix this now?

Reply:
![image](https://user-images.githubusercontent.com/12476526/84550294-0d4b6f80-acbf-11ea-990c-4e02fc67eb11.png)
Since -f is just a flag and doesn't need any value. We can pass null here and it will force push. I think the behavior the original comment is explaining is actually expected based on the documentation.